packages/dapp/src/components/Dashboard/ActivityListItem.tsx

```diff
@@ -83,7 +61,6 @@ class ActivityListItemComponent extends React.Component<
       const buttonTextTuple = this.getButtonText();
       const props = {
         newsroomName: newsroomData.name,
-        charter,
```

Comment:
The changes in this file seem to be unrelated to the PR and I think are reverting recent updates. The removal of charter stuff in particular will definitely break things.